src/test/java/net/sf/jabref/exporter/ExportFormatTest.java

```diff
@@ -23,75 +24,68 @@
 import net.sf.jabref.model.database.BibDatabase;
 import net.sf.jabref.model.entry.BibEntry;
 
+@RunWith(Parameterized.class)
 public class ExportFormatTest {
 
+    public IExportFormat exportFormat;
+    public String exportFormatName;
+    public static BibDatabase database;
+    public static Charset charset;
+    public static MetaData metaData;
+    public static List<BibEntry> entries;
+
+    public ExportFormatTest(IExportFormat format, String name) {
+        exportFormat = format;
+        exportFormatName = name;
+    }
+
+
     @Rule
     public TemporaryFolder testFolder = new TemporaryFolder();
 
 
-    @Before
-    public void setUp() {
-        Globals.prefs = JabRefPreferences.getInstance();
-        ExportFormats.initAllExports();
+    @BeforeClass
```

Comment:
I think before is preferable to beforeclass since then every test gets a fresh database and the potentially modifications of the previous test do not influence the next test.